client/landing/gutenboarding/components/domain-picker/style.scss

```diff
@@ -8,14 +10,37 @@
 	}
 }
 
-.domain-picker__choose-domain-header,
+.domain-picker__choose-domain-header {
+	font-weight: bold;
+	text-align: center;
+	margin-bottom: 20px;
+}
 .domain-picker__recommended-header {
 	font-weight: bold;
 }
 
-.domain-picker {
-	.components-panel__row {
-		flex-direction: column;
-		align-items: stretch;
+.domain-picker__panel-row {
+	flex-direction: column;
+	align-items: stretch;
+}
+
+.domain-picker__suggestion-item {
+	display: flex;
+	justify-content: space-between;
+	margin-top: 18px;
+}
+
+.domain-picker__suggestion-action {
+	color: var( --studio-blue-40 );
+}
+
+.domain-picker__has-domain {
+	align-items: center;
+	.components-button {
+		color: var( --studio-blue-30 );
```

Comment:
We were already using some of this Calypso variables. We can replace them when all designs will be made with the Gutenberg color scheme.